Client.xcodeproj/project.pbxproj

```diff
@@ -14488,6 +14489,7 @@
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = 43AQ936H96;
 				INFOPLIST_FILE = Client/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 14.0;
```

Comment:
Wondering why do you need this as this it should get from the common's file? 🤔

Reply:
Without it build was failing

Reply:
This is very strange, what about the code on the epic branch. Does that build?